libs/components/atoms/src/lib/empty-biomarker/empty-biomarker.component.ts

```diff
@@ -0,0 +1,31 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+
+/**
+ *  hra empty biomarker Component
```

Comment:
Improve documentation. Documentation needs to be descriptive and read like a book pretty much